src/environments/reyserve.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {ServerOptions} from 'polyserve/lib/start_server';
+import {Target, TargetImpl} from '../target/target';
+import * as temp from 'temp';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as child_process from 'child_process';
+
+temp.track();
+
+const REYSERVE_SOURCES = '/Users/ajo/workspace/reyserve';
+const REYSERVE_FILES = ['app.yaml', 'reyserve.py', 'http2push.py'];
+
+export class ReyServe implements TargetImpl {
+  serve(options: ServerOptions): Promise<any> {
+    return new Promise((resolve, reject) => {
+    // Make temp dir
```

Comment:
to expand: I think most of the work here should be in `ReyServe.build`, and serve should make sure build has been called (maybe we call it for every serve for now), and then just call dev_appserver.py